ide/static/js/canvas.js

```diff
@@ -235,7 +236,14 @@ class Canvas extends React.Component {
   allowDrop(event) {
     event.preventDefault();
   }
+  clickMultipleLayerEvent(event, layerId) {
+    event.preventDefault();
+    if (event.target.tagName.toLowerCase() == 'div') {
+      this.props.addLayerToMultipleSelection(layerId);
```

Comment:
Webpack error:
`242:18  error  'addLayerToMultipleSelection' is missing in props validation  react/prop-types`